.github/workflows/pr-lint.yaml

```diff
@@ -0,0 +1,33 @@
+name: lint pr
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
(Comment/Opinion)

I didn't expect this syntax to be mandatory.
* It may discourage external contributors.
* It may break scripts and automated PRs.

It's not possible to make this more like a warning/notification right?

Reply:
> more like a warning

No, at least not per the docs. I checked that it works with Dependabot, but Release PRs would currently fail. Will update the script. For external contributors it should be fine, because it is easy to fix and only occurs after a PR is already raised, i.e. when most of the work is already done.